app/index.html

```diff
@@ -247,8 +247,10 @@ <h1>About Myself</h1>
       </div>
       <div class="card-description about-body">
         <p>
-          Hello I'm a software developer! I can help you build a product,
```

Comment:
1. There should be  2 blocks The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right as in the desgin.
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/e90fb807-a49a-4c1f-8728-652caca6683a)
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/9997122a-e995-4e61-b7d3-844be33aaf73)

- [ ] Please make sure that the margins and alignment of the about myself section matches the design 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/754ae5a4-e35f-477f-bb86-6367d3d2cd3f)
Figma 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/e1a65a39-d811-481f-9f5b-37017c141a79)